packages/xo-server-test/src/backupNg/backupNg.spec.js

```diff
@@ -366,7 +369,7 @@ describe('backupNg', () => {
   })
 
   test('execute three times a rolling snapshot with 2 as retention & revert to an old state', async () => {
-    jest.setTimeout(6e4)
+    jest.setTimeout(18e4)
```

Comment:
Why this change?

Reply:
I revert this change because it's relative to the test environment (network, RAM, CPU ...).